docker/docker-static-build.sh

```diff
@@ -11,7 +11,7 @@ set -euo pipefail
 binaries=("tezos-admin-client" "tezos-client" "tezos-node" "tezos-signer" "tezos-codec")
 
 for proto in $(jq -r ".active | .[]" ../protocols.json); do
-    binaries+=("tezos-accuser-$proto" "tezos-baker-$proto" )
+    binaries+=("tezos-accuser-$proto" "tezos-baker-$proto" "tezos-tx-rollup-node-$proto" "tezos-tx-rollup-client-$proto")
```

Comment:
This looks good, but I believe that you missed adding new entries to `nix/build/release-binaries.nix`